modules/light-clients/08-wasm/client/cli/tx.go

```diff
@@ -53,3 +59,65 @@ func newStoreCodeCmd() *cobra.Command {
 
 	return cmd
 }
+
+// newStoreCodeGovCmd returns the command to send a proposal to store new wasm bytecode.
+func newStoreCodeGovCmd() *cobra.Command {
```

Comment:
I think this aligns more with the naming we're using in other gov CLIs.

```suggestion
func newSubmitStoreCodeProposalCmd() *cobra.Command {
```

Reply:
And we also need to do the trick to add this in `txCmd.AddCommand`, right? :)